README.md

```diff
@@ -80,6 +80,16 @@ rspec:
     - let_it_be
 ```
 
+If you have your own custom `example`-like methods like `it`, you can add them to your `.solargraph.yml` file like this:
+
+```yaml
+# .solargraph.yml
+# ...
+rspec:
+  example_methods:
+    - my_it
+```
```

Comment:
**minor**: It would be helpful to hint, where this can be useful, ie. [rspec-given](https://github.com/jimweirich/rspec-given) mention

Reply:
Sounds good! I've added it

Reply:
TY!